### DISCUSSION TITLE
English version

### DISCUSSION THREAD

**Discussion by mglavind** [AK-Homberger/NMEA2000-Workshop#3] [Ideas]
I seem to keep getting in on your amazing work about DIY solutions to NMEA2K and tinkering with ESP32, and I keep thinking about why I didn't listen more during my german classes back in high school. If you have any interest in broadening this to an English version, I would love to help all I can (but my german is not the best).

**Comment by AK-Homberger** [comment]
From my perspective Google translate is doing a good job here. Did you find critical errors in the translation?
